Manager.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+namespace AirportTicketBooking
+{
+    internal class Manager : User
+    {
+        private string Password;
+
+        public void SetPassword(string password)
+        {
+            Password = password;
+        }
+
+        public bool CheckPassword(string password)
```

Comment:
it's more like validation, let's create ManagerValidatior Class and put this one there
 it's an ambiguous name as well, check the password if what?